src/main/java/io/openliberty/tools/intellij/util/Constants.java

```diff
@@ -94,4 +94,21 @@ public final class Constants {
     public static final String JAKARTA_LANG_SERVER="Eclipse LSP4Jakarta";
     public static final String MICROPROFILE_SERVER="Eclipse LSP4MP";
 
+    /**
+     * Constants for Integration testing
+     */
+    public static final String SYSTEM_RESOURCE_JAVA = "SystemResource.java";
+    public static final String SYSTEM_RESOURCE_2_JAVA = "SystemResource2.java";
+    public static final String SYSTEM_RESOURCE = "SystemResource";
+    public static final String SYSTEM_RESOURCE_2 = "SystemResource2";
+
+    public static final String SERVER_XML = "server.xml";
+    public static final String SERVER_ENV = "server.env";
+    public static final String BOOTSTRAP_PROPERTIES = "bootstrap.properties";
+    public static final String COMPACT_MODE = "Compact Mode";
+
+    public static final String SERVICE_LIVE_HEALTH_CHECK_JAVA = "ServiceLiveHealthCheck.java";
+    public static final String MPG_PROPERTIES = "microprofile-config.properties";
+
+    public static final String CLOSE_ALL_TABS = "Close All Tabs";
```

Comment:
@anusreelakshmi934 , based on Trevor suggestion, since constants are only used for the tests, then they should go in a file under the test directory.

Reply:
I have modified it. Please re review.